### DISCUSSION TITLE
NotImplementedError: WARNING ⚠️ 'YOLO' model does not support '_new' mode for 'None' task yet.

### DISCUSSION THREAD

**Discussion by zengqing1218** [ultralytics/ultralytics#13190] [Q&A]
I tried to add a new module in "ultralytics" to improve "yolov8 world", but the above error occurred. How can I solve it

**Comment by pderrenger** [comment]
@zengqing1218 hello! It looks like you're trying to add a new module to the YOLOv8 World model, but encountered a `NotImplementedError`. This error typically occurs when the code attempts to execute a function or mode that hasn't been defined or supported yet.

To resolve this issue, please ensure that the new module or function you're adding is properly integrated and that all necessary methods are implemented correctly. If you're adding a new mode, you'll need to define how this mode operates within the existing framework.

If you need further guidance, please provide more details about the module you're adding and the specific functionalities you're trying to implement. This will help us assist you more effectively. Thank you for contributing to the Ultralytics project! 🚀
